legal-api/src/legal_api/models/resolution.py

```diff
@@ -88,3 +88,13 @@ def find_by_type(cls, business_id: int, resolution_type: str):
             filter(Resolution.resolution_type == resolution_type). \
             all()
         return resolutions
+
+    @classmethod
+    def find_latest_for_business(cls, business_id: int, resolution_type: str) -> Resolution:
+        """Return the latest resolution of a specific type for a business."""
+        resolution = db.session.query(Resolution). \
+            filter(Resolution.business_id == business_id). \
+            filter(Resolution.resolution_type == resolution_type). \
+            order_by(Resolution.id.desc()). \
```

Comment:
Can resolutions ever be out of order? Eg, the latest ID isn't the latest date?

Reply:
it suppose not I think, but I just removed this function, it's unused for now.